app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ProjectsController < ApplicationController
+  before_action :set_project, only: %i[ show edit update destroy ]
+
+  def index
+    @projects = Project.all
+  end
+
+  def show
+    @tasks = @project.tasks.order(id: :desc)
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def edit
+  end
+
+  def create
+    @project = Project.create! project_params
+
+    redirect_to project_url(@project), notice: "Project was successfully created."
```

Comment:
is there a way to display these flash messages with morphing on? send them via turbostream somehow? thanks!

Reply:
@ahlucas for sure. I'm not rendering the flash in the demo screens but if I were, they would show up!

Reply:
@jorgemanrubia thanks for your comment! even with morphing on though? is there a mechanism implemented for this? if i do for example in my templates something like this--

```
- if flash.any?
  ul.flashes
    - flash.each do |type, message|
      li class=type
        span
          == message
```

it updates/populates the "flash" variable via turbo stream actions? with just `notice: "Project was successfully created."` addition to controller redirect?

currently struggling with this. thanks so much!

Reply:
@ahlucas I can confirm that it works, but you need to add an unique id for your flash container.
Here's an example: https://github.com/thdaraujo/turbo-8-morphing-demo/pull/2/files

For a turbo stream, you could have your turbo streams inherit from a turbo_stream layout (as mentioned [here](https://evilmartians.com/chronicles/hotwire-reactive-rails-with-no-javascript#:~:text=UPDATE%20(2021%2D04%2D13)%3A%20Alex%20Takitani%20suggested%20a%20more%20elegant%20solution)).